packages/fx-core/resource/yaml-schema/v1.5/yaml.schema.json

```diff
@@ -72,7 +72,8 @@
           { "$ref": "#/definitions/teamsAppCopyAppPackageToSPFx" },
```

Comment:
make sure `packages/fx-core/resource/yaml-schema/yaml.schema.json` is also synced.